### PR TITLE
@craigspaeth => Delete non-partner/non-fair articles

### PR DESCRIPTION
**Pull request by kanaabe**
- Removes /article/slug pages
- Fair articles now have slugs like `/fair/article/slug` instead of `/article/slug`
- Partner articles mostly stay the same (`/partner/article/slug`) -- the breadcrumb is moved to the partner app. 
- Clean up the article component by removing sections that are only for editorial articles
- Clean up `fetchRelated` logic in the article model
- Remove section support and only allow the Venice Biennale section. 

Soooo close to 2K deletion!

This is likely VERY hard to read and I apologize for that! There wasn't much I could do but I tried to keep the commit messages true so please read from there. 

cc @eessex

### CODE REVIEW

File: apps/articles/routes.coffee [hunk 1]
```diff
@@ -13,62 +13,39 @@ embed = require 'embed-video'
...
-  return next() if req.params.id.match(EOY_2016_SLUGS)?.length
+  # Handles fair and partner articles
   article = new Article id: req.params.id
   article.fetch
-    cache: true
     error: -> next()
     success: =>
       article.fetchRelated
         success: (data) ->
-          res.locals.sd.ARTICLE = article
```
Comment (craigspaeth) [inline apps/articles/routes.coffee:24]: 😀

File: apps/articles/routes.coffee [hunk 2]
```diff
@@ -13,62 +13,39 @@ embed = require 'embed-video'
...
-            featuredSection: data.section
-            featuredSectionArticles: data.sectionArticles if data.section
-            relatedArticles: data.relatedArticles
-            calloutArticles: data.calloutArticles
-            superArticle: data.superArticle
-            embed: embed
-            jsonLD: stringifyJSONForWeb(article.toJSONLD())
-            videoOptions: { query: { title: 0, portrait: 0, badge: 0, byline: 0, showinfo: 0, rel: 0, controls: 2, modestbranding: 1, iv_load_policy: 3, color: "E5E5E5" } }
-            lushSignup: true
+          if data.partner
```
Comment (craigspaeth) [inline apps/articles/routes.coffee:23]: 👌

File: apps/articles/routes.coffee [hunk 3]
```diff
@@ -13,62 +13,39 @@ embed = require 'embed-video'
...
+          else if data.fair
+            return res.redirect "/#{data.fair.get('default_profile_id')}/article/#{article.get('slug')}"
+          else return next()
 
 module.exports.redirectPost = (req, res, next) ->
   res.redirect 301, req.url.replace 'post', 'article'
 
 module.exports.section = (req, res, next) ->
-  new Section(id: req.params.slug).fetch
+  new Section(id: 'venice-biennale-2015').fetch
```
Comment (craigspaeth) [inline apps/articles/routes.coffee:33]: 👍

File: apps/fair/routes.coffee [hunk 4]
```diff
@@ -91,6 +95,29 @@ module.exports.articles = (req, res, next) ->
         profile: req.profile
     error: res.backboneError
 
+module.exports.article = (req, res, next) ->
```
Comment (craigspaeth) [inline apps/fair/routes.coffee:98]: 👍

File: apps/fair/test/templates.coffee [hunk 5]
```diff
@@ -272,4 +273,26 @@ describe 'Search', ->
         , params)
       render {
         shows: new Shows [fabricate 'show', fair_location: null]
-      }, 'artist'
\ No newline at end of file
+      }, 'artist'
+
+describe 'article page template ', ->
+
+  beforeEach ->
```
Comment (craigspaeth) [inline apps/fair/test/templates.coffee:280]: Orphaned beforeEach here

File: config.coffee [hunk 6]
```diff
@@ -69,7 +69,7 @@ module.exports =
   MARKETING_SIGNUP_MODAL_COPY: 'Sign up for early access'
   MARKETING_SIGNUP_MODAL_IMG: 'http://placekitten.com/200/200'
   MARKETING_SIGNUP_MODAL_SLUG: 'miami'
-  EOY_2016_SLUGS: ''
+  FAIR_CHANNEL_ID: '5759e4f3b5989e6f98f77998'
```
Comment (craigspaeth) [inline config.coffee:72]: 👍

File: models/article.coffee [hunk 7]
```diff
@@ -42,74 +44,35 @@ module.exports = class Article extends Backbone.Model
   shareDescription: ->
     (@get('share_description') or @get('thumbnail_title')) + " @artsy"
 
+  isFairArticle: ->
```
Comment (craigspaeth) [inline models/article.coffee:47]: 👍

File: models/article.coffee [hunk 8]
```diff
@@ -42,74 +44,35 @@ module.exports = class Article extends Backbone.Model
...
+    # associated to a fair and the Fairs team has written it
+    @get('fair_ids')?.length > 0 and @get('channel_id') is sd.FAIR_CHANNEL_ID
+
   fetchRelated: (options) ->
     Articles = require '../collections/articles.coffee'
     dfds = []
-    relatedArticles = new Articles()
```
Comment (craigspaeth) [inline models/article.coffee:48]: 😮  nice cleanup!

**Review by craigspaeth — APPROVED** [review]

**Comment by craigspaeth** [comment]
Wohoo!! 👏  nearly -2k LOC. Looks good to me, I don't see any blocking issues skimming through the diff. Feel free to merge when 💚

**Comment by kanaabe** [comment]
Bye bye MG! 👋  I'll do one last test run at home and then merge. Thanks @craigspaeth!

**Pull request merged**